packages/components/src/Chips/ChipDismissible.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { SyntheticEvent } from "react";
+import { InternalChip, InternalChipProps } from "./InternalChip";
+import { InternalChipButton } from "./InternalChipButton";
+
+interface ChipDismissibleProps
+  extends Pick<
+    InternalChipProps,
+    "label" | "disabled" | "invalid" | "prefix" | "onClick"
+  > {
+  onRequestRemove(event: SyntheticEvent<HTMLDivElement>): void;
+}
+
+export function ChipDismissible({
+  label,
+  disabled,
+  invalid,
+  prefix,
+  onClick,
+  onRequestRemove,
+}: ChipDismissibleProps) {
+  return (
+    <InternalChip
+      label={label}
+      disabled={disabled}
+      invalid={invalid}
+      prefix={prefix}
+      onClick={onClick}
+      suffix={
+        <InternalChipButton
+          icon="remove"
+          label={label}
+          invalid={invalid}
+          disabled={disabled}
+          onClick={onRequestRemove}
```

Comment:
Hmmm. it shouldn't. the remove button has an internal thing that stops the click from bubbling. anyhoo, TS wouldn't allow that tho since `onRequestRemove` is required

Reply:
Noticed this while testing for accessibility!

Reply:
oh, i'll look in to that

Reply:
Should be fixed now. needed to prevent default on key down. was just preventing bubbling on key up